alf/algorithms/particle_vi_algorithm.py

```diff
@@ -38,8 +38,8 @@ class ParVIAlgorithm(Algorithm):
 
         1. Stein Variational Gradient Descent (SVGD):
 
-        Feng et al "Learning to Draw Samples with Amortized Stein Variational
-        Gradient Descent" https://arxiv.org/pdf/1707.06626.pdf
+        Liu, Qiang, and Dilin Wang. "Stein Variational Gradient Descent: 
```

Comment:
Need to indent two more chars to make the format correct.

Reply:
Updated here and some other docstrings.